third-party/realdds/doc/DDS ICD.md

```diff
@@ -0,0 +1,147 @@
+    "options":            //Array of supported options, the size of n-options
+    [
+        {"description":"Enable/Disable global timestamp","name":"Global Time Enabled","range-default":1.0,"range-max":1.0,"range-min":0.0,"range-step":1.0,"value":1.0},
+        {"description":"HDR Option","name":"Hdr Enabled","range-default":0.0,"range-max":1.0,"range-min":0.0,"range-step":1.0,"value":0.0}
+    ]
+
+#### stream-header format example
+
+    "default-profile-index":0,
+    "id":"stream-header",
+    "name":"Infrared 1",
+    "profiles":           //All stream profiles. Each profile is an array of frequency, format, width, height.
+    [
+        [25,"Y16",1288,808],[15,"Y16",1288,808],[30,"GREY",1280,720],[15,"GREY",1280,720],[5,"GREY",1280,720],[90,"GREY",848,480],[60,"GREY",848,480],[30,"GREY",848,480],[15,"GREY",848,480],[5,"GREY",848,480],[90,"GREY",640,480],[60,"GREY",640,480],[30,"GREY",640,480],[15,"GREY",640,480],[5,"GREY",640,480],[90,"GREY",640,360],[60,"GREY",640,360],[30,"GREY",640,360],[15,"GREY",640,360],[5,"GREY",640,360],[90,"GREY",480,270],[60,"GREY",480,270],[30,"GREY",480,270],[15,"GREY",480,270],[5,"GREY",480,270],[90,"GREY",424,240],[60,"GREY",424,240],[30,"GREY",424,240],[15,"GREY",424,240],[5,"GREY",424,240]
+    ],
+    "sensor-name":"Stereo Module",
+    "type":"ir"
+
+#### stream-options format example
+
+    "id":"stream-options",
+    "n-options":3,        //Integer, number of device options
+    "options":            //Array of supported options, the size of n-options
+    [
+        {"description":"Enable / disable backlight compensation","name":"Backlight Compensation","range-default":0.0,"range-max":1.0,"range-min":0.0,"range-step":1.0,"value":0.0},
+        {"description":"UVC image brightness","name":"Brightness","range-default":0.0,"range-max":64.0,"range-min":-64.0,"range-step":1.0,"value":0.0},
+        {"description":"UVC image contrast","name":"Contrast","range-default":50.0,"range-max":100.0,"range-min":0.0,"range-step":1.0,"value":50.0}
+    ],
+    "stream-name":"Infrared 1"
+
+### Instant notifications
+
+Other notifications are sent as needed, as a reply to **set-option** or **query-option**
+
+#### query-option reply format example
+
+    "id":"query-option",
+    "counter": 123,       //Integer, identical to the set-option command counter
+    "value": 12.34,       //Float
+    "successful":false,
+    "failure-reason":""
+
+#### set-option reply format example
+
+    "id":"set-option",
+    "counter": 123,       //Integer, identical to the set-option command counter
+    "successful":false,
+    "failure-reason":""
+
+## Control
+
+Controls are commands from the user to the camera.
+Using controls the user can open or close streaming, or he can query current option values or set new values affecting the camera behaviour.
+
+When using librealsense API commands will automatically be converted to the appropriate commands that will be sent to the camera over the DDS network.
```

Comment:
When using librealsense API commands, "these" will automatically - these is missing here

Reply:
I don't think that "these" is needed here. I meant that API command like `sensor.open( profile )` will be converted to `open-streams` DDS command. I will add examples

Reply:
I understood the meaning of the sentence, but there is some syntax error here. Let's talk about it.